src/network/network.cpp

```diff
@@ -1329,8 +1423,16 @@ void Network::Server_Send_AUTH(NetworkConnection& connection)
 	}
 }
 
-void Network::Server_Send_MAP(NetworkConnection* connection)
+void Network::Server_Send_MAP(NetworkConnection * connection, bool resync, sint16 viewX, sint16 viewY, sint16 viewZoom, sint16 viewRotation)
 {
+	// pause the game if not already
+	bool pause = false;
+	if (RCT2_GLOBAL(RCT2_ADDRESS_GAME_PAUSED, uint32) & 0) {
```

Comment:
Why does the game need to be paused when someone connects? I'm worried about when someone who just wants to ruin the game for everyone joins a server multiple times to make it keep pausing and unpausing.